internal/runners/clean/cache.go

```diff
@@ -109,3 +128,19 @@ func (c *Cache) removeProjectCache(projectDir, namespace string, force bool) err
 
 	return nil
 }
+
+func (c *Cache) checkPathInUse(path string) (bool, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), model.SvcTimeoutMinimal)
```

Comment:
I don't think we should have such an aggressive timeout here. Especially for process in use checks, which on Windows tend to be slow. Do we bubble up timeout errors or that would just result in no processes in use being asserted?

Reply:
Good point. I've increased it a fair amount but let me know if we should do further adjustments. 

Looking at the client code the error from the context timeout _should_ be bubbled up as a `context deadline exceeded`. Won't be super useful to the user... we could maybe catch that specific error and continue with the removal if we encounter it.